hrjobcontract/gulp-tasks/sass.js

```diff
@@ -0,0 +1,14 @@
+var exec = require('child_process').exec;
+
+module.exports = function () {
+  return {
+    // @NOTE this will replace the original "sass:main" task, as job contract
+    // needs a completely different logic for compiling sass
+    main: function (done) {
+      exec('compass compile', { cwd: __dirname + '/../' }, function (_, stdout, stderr) {
```

Comment:
I don't think we should use '_', is that an event param? Better to use 'event' param, it will be skipped by semistandard I think.